mysqli/mysqli.php

```diff
@@ -918,7 +930,7 @@ class mysqli_result implements IteratorAggregate
     #[LanguageLevelTypeAware(['8.1' => 'int'], default: '')]
     public $field_count;
     /**
-     * @var array
+     * @var int[]|null
```

Comment:
Shouldn't be just `?array`? That's a more safe way of hinting this. See also https://github.com/php/php-src/blob/master/ext/mysqli/mysqli.stub.php#L359

Reply:
I can simply be `?array` but I just copied it over from the procedural variant https://github.com/kamil-tekiela/phpstorm-stubs/blob/a8c609c8ad9e56e34967252d024b36aec13dc401/mysqli/mysqli.php#L1672
Which one is better? I think it should be one or the other in both circumstances.

Reply:
I believe this one should be left as `?array` or `array|null`, because procedural `mysqli_fetch_lengths()` is described separately & `$lengths` is more of an OOP context.